src/SprykerEco/Shared/Computop/Config/ComputopApiConfig.php

```diff
@@ -146,4 +146,6 @@ interface ComputopApiConfig
     public const PSD2_MSG_VERSION = '2.0';
     public const UNSCHEDULED_CUSTOMER_INITIATED_TRANSACTION = 'CIT';
     public const NOTIFICATION_PARAMETER_PAYMENT_TYPE = 'type';
+    public const TAX_TOTAL = 'TaxTotal';
```

Comment:
Public constants are part of our public API, therefore they should be tagged with the `@api` doc block tag and provided with specifications.

Reply:
Added for new constants.